PublishOpenDebugAD7.sh

```diff
@@ -132,4 +132,5 @@ fi
 log_and_exec_cmd "dotnet build ${__ScriptDirectory}/src/MIDebugEngine-Unix.sln -c ${__Configuration}"
 log_and_exec_cmd "dotnet publish ${__ScriptDirectory}/src/OpenDebugAD7/OpenDebugAD7.csproj ${__DotnetPublishArgs}"
 log_and_exec_cmd "cp ${__ScriptDirectory}/bin/${__Configuration}/Microsoft.MIDebugEngine.dll ${__OutputFolder}/."
-log_and_exec_cmd "cp ${__ScriptDirectory}/bin/${__Configuration}/Microsoft.MICore.dll ${__OutputFolder}/."
\ No newline at end of file
+log_and_exec_cmd "cp ${__ScriptDirectory}/bin/${__Configuration}/Microsoft.MICore.dll ${__OutputFolder}/."
+log_and_exec_cmd "cp ${__ScriptDirectory}/bin/${__Configuration}/Microsoft.VisualStudio.Debugger.Interop.DAP.dll ${__OutputFolder}/."
```

Comment:
Since `Microsoft.VisualStudio.Debugger.Interop.DAP` is already a reference in `OpenDebugAD7.csproj`, you can have it copied if you add `<Private>true</Private>` in the `<ProjectReference>...</ProjectReference>` instead of forcing a copy in the publish script. 

Reference:
![image](https://user-images.githubusercontent.com/3953714/126386507-cc1715fc-0037-42b0-a642-c58cb6a5d9d8.png)

https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-items?view=vs-2019#projectreference

Reply:
You shouldn't need to copy it - it is embedded

Reply:
I saw your comment about getting the build error with ambiguous types. I don't see it when I build MIEngine on my own box. Where are you seeing the problem? I rebased your changes to make it easier to view: [gregg-miskelly/completion](https://github.com/gregg-miskelly/MIEngine/tree/completion).

Reply:
When trying to evaluate `m_engine is IDebugProgramDAP` in the debug console, as it is done in the code.

Reply:
Are you saying that everything actually works, but the debugger's expression evaluator gets confused?

Reply:
(Like at runtime, the capability gets correctly assigned, but if you try to evaluate `m_engine is IDebugProgramDAP` in the debugger, the debugger is confused which definition of `IDebugProgramDAP` to use)

Reply:
No the expression is also false at runtime so the whole feature gets disabled.

Reply:
What .NET Runtime are you using? (.NET 5, Mono, .NET Framework)
What OS?

Reply:
```sh
$ tail ~/.vscode-server/extensions/ms-vscode.cpptools-*/debugAdapters/OpenDebugAD7
#ARGS+=("--pauseForDebugger")
dotnet "${BIN_DIR}/bin/OpenDebugAD7.dll" "${ARGS[@]}" $*
$ dotnet --version
5.0.301
```
WSL2, Ubuntu 21.04

Reply:
Okay, I don't know why you are seeing that. I will set it up and see if I can repro.

Reply:
I can reproduce it. Currently investigating why the [embedded type equivalence](https://docs.microsoft.com/en-us/dotnet/framework/interop/type-equivalence-and-embedded-interop-types#marking-com-types-for-type-equivalence) is not working in this scenario.